lib/ontologies_api_client/request_federation.rb

```
@@ -22,14 +22,17 @@ def federated_get(params = {}, &link)
 
             unless portal_status
+              HTTP.log("Error in federation #{portal_name} is down status cached for 10 minutes")
               next [OpenStruct.new(errors: "Problem retrieving #{portal_name}")]
             end
 
             main_thread_locals.each { |key, value| Thread.current[key] = value }
             begin
-              HTTP.get(link.call(conn.url_prefix.to_s.chomp('/')), params, connection: conn)
+              portal_params = params[portal_name.to_s.downcase] || params
+              HTTP.get(link.call(conn.url_prefix.to_s.chomp('/')), portal_params, connection: conn)
             rescue Exception => e
```

RuboCop (1.75.5)

[warning] Useless assignment to variable - `e`.

(Lint/UselessAssignment)